portfolio/src/main/webapp/script.js

```diff
@@ -14,8 +14,15 @@
 
 async function getFormMessage() {
   const response = await fetch('/data');
-  const quote = await response.text();
-  document.getElementById('quote-container').innerHTML = quote;
+  const jsonComments = await response.text();
+  var comments = JSON.parse(jsonComments);
+  
+  Object.keys(comments).forEach(function(commentIndex){
```

Comment:
Use `for(const key in obj) {...}` for iterating over objects, and `for(const item of items) {...}` for iterating over arrays.

https://www.w3schools.com/jsref/jsref_forin.asp

Reply:
done! whoop! 🙌